generators/client/files-angular.js

```diff
@@ -545,8 +545,12 @@ module.exports = {
     files
 };
 
-function writeFiles() {
+function writeFiles(calledByBlueprint) {
     mkdirp(MAIN_SRC_DIR);
     // write angular 2.x and above files
-    this.writeFilesToDisk(files, this, false, 'angular');
+    if (calledByBlueprint) {
+        this.writeFilesToDisk(files, this, false, '../../../node_modules/generator-jhipster/generators/client/templates/angular');
```

Comment:
This would cause issues when linking and stuff. Let me try something out for this tomorrow. I have some ideas on how to fix this

Reply:
i'll commit my updates to this PR

Reply:
> I have some ideas on how to fix this

So do you confirm I'm not crazy and blueprint generator currently doesn't work ?
Because I tried to do `link` :
- in my local, with NPM / Yarn
- in a Docker container, with NPM / Yarn
- in Travis, with NPM / Yarn
I always have the issue when using `super._writing()` : files are not found in the generated blueprint.

Reply:
Haha, it was working during and before the conference, check my sample blueprint it has similar calls on the entity server, but then with all the NPM/Yarn linking confusions, I'm not sure anymore. SO tomorrow i'm gonna really tests the shit out of this :wink: 

Btw I always link the blueprint globally using `npm link` and not locally on the app folder. Not sure if that matters (yarn link doesnt work well for me, as yeoman doesnt see the generator)

Anyway let me figure this out tomorrow